src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
```

Comment:
You shouldn't need a spring.factories file unless this project was a library dependency.  Spring should find the @ Configuration class.

Reply:
removed